### PR TITLE
Removes PVL version restriction

### PR DESCRIPTION
**Pull request by acpaquette**
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

Related #11

### CODE REVIEW

**Comment by jessemapel** [comment]
This is not true for PLIO 1.2.4. The PVL requirement change should happen on the next release PR, not separately.

**Comment by acpaquette** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ requirements:
   run:
     - python
     - numpy
-    - pvl <1.0
+    - pvl
```
Comment (jlaura) [inline recipe/meta.yaml:25]: Should we specify a range here? Is PVL using semantic versioning where we can use a >= 1.0, <= 2.0 or do we need to do something else?

Comment (jessemapel) [reply recipe/meta.yaml:25]: I think we need to specify >= 1.0

Comment (jlaura) [reply recipe/meta.yaml:25]: Should this change happen upstream (in our repo) for the meta and the environment and conda-forge will pull that change or should do we need to make the change in both repos. I can't remember if conda-forge updates the recipe if the upstream recipe updates.

**Review by jessemapel — APPROVED** [review]
Glad to see the linter is still getting its cut.

**Pull request merged**